features/doc-document.feature

```diff
@@ -8,3 +8,11 @@ Feature: Document properties and methods
      When I assign bookmark = document.start_bookmark("Target")
      Then bookmark.name == "Target"
       And bookmark.id is an int
+
+  @wip
+  Scenario: Document.end_bookmark()
+    Given a Document object as document
+     When I assign bookmark = document.start_bookmark("Target")
+      And I end bookmark by calling document.end_bookmark(bookmark)
+      And I assign bookmark = document.bookmarks.get("Target")
+     Then bookmark.name == "Target"
```

Comment:
What is this testing exactly? Is it that a bookmark can't be looked up by name until it is closed ("end"ed)?

I'm supposing that's the only real way we have of telling (via the API) that `.end_bookmark()` did its job. If that's the case, this is fine. I would combine the last two lines though to remove the mutation of the bookmark variable (from its initial value) so the last line would be:

```cucumber
Then document.bookmarks.get("Target").name == "Target"
```

This avoids the fact that line 18 would resolve `True` if you inserted it immediately below line 15.

Also I think it's worth a comment like `# ---bookmark can only be looked up by name if it is closed---` just so it's clear to future readers what the strength of the test depends on; we're taking "lookup-ability" as a proxy for "closed".

Reply:
Fixed